.gitignore

```diff
@@ -5,4 +5,5 @@ lcm_types/*.hpp
 lcm_types/*.py
 __pycache__/
 python_scripts/python_types/*
-python_scripts/logs/*
\ No newline at end of file
+python_scripts/logs/*
+.vscode*
```

Comment:
whats the logic behind this change?

Reply:
this one likely shouldnt have been pushed. 
their are vscode files I didnt want to see